Ingredients/Startup.cs

```diff
@@ -51,19 +57,22 @@ public async Task Configure(WebApplication app, IWebHostEnvironment env, IServic
         {
             var driver = serviceProvider.GetRequiredService<IDriver>();
             var ingredientsRepository = serviceProvider.GetRequiredService<IIngredientsRepository>();
+            var allergensRepository = serviceProvider.GetRequiredService<IAllergensRepository>();
+            var categoriesRepository = serviceProvider.GetRequiredService<ICategoriesRepository>();
             var insertDataCSV = new InsertDataCSV(driver);
-            await insertDataCSV.InsertDataFromCsv(ingredientsRepository, "ingredients.csv");
+           insertDataCSV.InsertDataFromCsv(ingredientsRepository, allergensRepository, categoriesRepository, "/csvfiles/ingredients.csv", "/csvfiles/Allergene.csv", "/csvfiles/Categories.csv").Wait();
```

Comment:
Why use `.Wait()` here? When awaiting Tasks you should generally use `await` instead.

Something like this:

`await insertDataCSV.InsertDataFromCsv(ingredientsRepository, allergensRepository, categoriesRepository, "/csvfiles/ingredients.csv", "/csvfiles/Allergene.csv", "/csvfiles/Categories.csv");`